Tests/scripts/secrets.py

```diff
@@ -0,0 +1,287 @@
+import io
+import os
+import re
+import math
+import json
+import string
+import PyPDF2
+import validate_files_structure
+
+# secrets settings
+# Entropy score is determined by shanon's entropy algorithm, most English words will score between 1.5 and 3.5
```

Comment:
Minor note here, i like to keep logical separation in CONSTs so that when reading/searching for something you will see it straight away, try to separate(One blank line) the ENTROPY_THRESHOLD is pseudo configurable, the regexs are definitely unique for the rest of the consts, and the lists are too